docs/user-management/saml/setup.md

```diff
@@ -33,12 +33,63 @@ The steps to configure the IdP vary depending on your chosen IdP. These are some
 * Create an app for n8n in your IdP.
 * Map n8n attributes to IdP attributes:
 
-	| Name | Name format | Value (IdP side) |
-	| ---- | ----------- | ---------------- |
-	| `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress` | URI Reference | User email       |
-	| `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/firstname`    | URI Reference | User First Name  |
-	| `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/lastname`     | URI Reference | User Last Name   |
-	| `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/upn`          | URI Reference | User Email       |
+| Value (IdP side) | Name format | Name |
```

Comment:
I reordered the columns so that the URLs come last, this makes it easier to read the table.